src/lib/utils.ts

```diff
@@ -588,6 +589,16 @@ export function countNumberOfBytes(data: string) {
   return stripHexPrefix(data).length / 2;
 }
 
+export function countSignificantBits(data = '0x0') {
```

Comment:
Why giving a default value here? Also isn't should have at least two nibbles to be a valid byte?

```suggestion
export function countSignificantBits(data = '0x00') {
```

Reply:
The reason was that otherwise the code couldn't be tested using javascript in node. I'll change it back.

Reply:
Removed.